src/projects/projects.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@nestjs/common';
+import { NotFoundException } from '@nestjs/common/exceptions';
+import { plainToInstance } from 'class-transformer';
+import { Req, Res } from 'src/dtos';
+import { ProjectsRepository } from 'src/repositories';
+
+@Injectable()
+export class ProjectsService {
+  constructor(private projectsRepository: ProjectsRepository) {}
+
```

Comment:
```suggestion
  constructor(private readonly projectsRepository: ProjectsRepository) {}
```

Apply, please, this change to the other constructors as well, <3

This will prevent any update to the service instance, someone might do it unintentionally.

Reply:
Good tip! 

Applied in https://github.com/VAIRIX/cvs-backend/pull/18/commits/40c947bb4ca61373ab84f35fa715bcb4537de7ac